packages/sources/nav-consulting/src/transport/fundId.ts

```diff
@@ -0,0 +1,44 @@
+import { Requester } from '@chainlink/external-adapter-framework/util/requester'
+import { getValidatorIds } from './authentication'
+import { AdapterError } from '@chainlink/external-adapter-framework/validation/error'
+interface Response {
+  FundName: string
+  GlobalFundID: number
+  FundEndDate: string
+  FundDailyAccountingStartDate: string
+  FundDailyAccountingLastAvailableDate: string
+  FundOfficialAccountingLastAvailableDate: string
+  PortfolioLastAvailableDate: string
+}
+
+export const getFundId = async (
```

Comment:
unlikely `fundId` will change, right? worth caching?

Reply:
We are fetching both the fundId and the date so we can not cache. I think naming is confusing here